coder_sniffer/DrupalPractice/Test/InfoFiles/CoreVersionRequirementUnitTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace DrupalPractice\Test\InfoFiles;
+
+use Drupal\Test\CoderSniffUnitTest;
+
+class CoreVersionRequirementUnitTest extends CoderSniffUnitTest
+{
+
+
+    /**
+     * Returns the lines where errors should occur.
+     *
+     * The key of the array should represent the line number and the value
+     * should represent the number of errors that should occur on that line.
+     *
+     * @param string $testFile The name of the file being tested.
+     *
+     * @return array<int, int>
+     */
+    protected function getErrorList(string $testFile)
```

Comment:
return type hint "array" is missing. Strange that this does not cause a fatal error during the test run.

Reply:
ah sorry, getting ahead of myself. This is only done in #78